docs/_src/api/pydoc/answer-generator.yml

```diff
@@ -5,7 +5,7 @@ loaders:
     ignore_when_discovered: ['__init__']
 processors:
   - type: filter
-    expression: not name.startswith('_') and default()
+    expression: 
```

Comment:
I think the entire line could be deleted, no need to keep the empty expression here. Doesn't matter too much though.

Reply:
Ah noted for next time! For now I think I will merge as is so that we don't introduce an error or get too slowed down with reviews